<launch>
    <node pkg="depthimage_to_laserscan" type="depthimage_to_laserscan" name="depthimage_to_laserscan" output="screen">
    <remap from="image" to="/zed2/zed_node/depth/depth_registered" />

    <!--激光扫描的帧id。对于来自具有Z向前的“光学”帧的点云，该值应该被设置为具有X向前和Z向上的相应帧。-->
    <param name="output_frame_id" value="/laser"/>
    <!--用于生成激光扫描的像素行数。对于每一列，扫描将返回在图像中垂直居中的那些像素的最小值。-->
    <param name="scan_height" value="50" />
    <!--返回的最小范围（以米为单位）。小于该范围的输出将作为-Inf输出。-->
    <param name="range_min" value="0.45" />
    <!--返回的最大范围（以米为单位）。大于此范围将输出为+ Inf。-->
    <param name="range_max" value="2.00" />
    </node>

</launch>
